src/gui/editorwidgets/qgsvaluerelationsearchwidgetwrapper.h

```diff
@@ -56,7 +56,9 @@ class GUI_EXPORT QgsValueRelationSearchWidgetWrapper : public QgsDefaultSearchWi
   public:
     explicit QgsValueRelationSearchWidgetWrapper( QgsVectorLayer* vl, int fieldIdx, QWidget* parent = 0 );
     bool applyDirectly() override;
+    QString expression();
```

Comment:
no I can't.... QgsSearchWidgetWrapper::expression and QgsWidgetWrapper::valid are not const.
shall I change them?